src/main/java/org/schemaspy/cli/CommandLineArguments.java

```diff
@@ -102,6 +102,15 @@ public class CommandLineArguments {
     )
     private boolean noImplied = false;
 
+    @Parameter(
+            names = {
+                    "--noorphans",
```

Comment:
Should this be `-noorphans` and `--no-orphans`

Reply:
I couldn't find any SchemaSpy standard for this, and existing parameters are inconsistent. I'm happy to change if there's some thought behind it that I've missed.

> `-noorphans`

In my mind, single hyphens exist because of convenience: Allowing commonly used parameters to be expressed with much less typing. So they should be single letters. In other words, not every parameter needs a short version.

What purpose would long options with single hyphens serve?

> `--no-orphans`

Most of SchemaSpy's multi-world parameters do _not_ use kebab case (e.g., `--dbhelp` and `--outputDirectory`) but some do (e.g., `--no-html` and `--no-implied`). Personally, I prefer the kebab version and can update if you want.

Reply:
There is an issue for the inconsistency with the command line args. Should we say kebab and then do breaking changes to make it consistent.

Same goes for the short syntax, `-no` would be no orphans if it doesn't collide with another argument.

Reply:
Sounds good for moving forward with this PR. Let's create a new issue for consistency/policy.

Reply:
There is an issue

Reply:
#1031